### PR TITLE
Add info about Democracy extrinsics for Ledger

### PR DESCRIPTION
**Pull request by salmad3**

### CODE REVIEW

File: docs/kusama-ledger.md [hunk 1]
```diff
@@ -197,6 +197,26 @@ want to stake using an account stored on a Ledger device.
 - Select "Start Nominating" or "Start Validating" to start nominating or validating.
 - Confirm the transaction on Apps and on the Ledger device.
 
+### Removing Expired Democracy Locks
+
+Since Ledger does not support batch transactions, you must issue several extrinsics manually when you
+want remove expired democracy locks on an account stored on a Ledger device.
```
Comment (laboon) [inline docs/kusama-ledger.md]:
```suggestion
want to remove expired democracy locks on an account stored on a Ledger device.
```

File: docs/kusama-ledger.md [hunk 2]
```diff
@@ -197,6 +197,26 @@ want to stake using an account stored on a Ledger device.
...
+
+- Go to https://polkadot.js.org/apps/#/accounts. Look at your expired locks and note which referenda
+  they were for.
+- Go to https://polkadot.js.org/apps/#/extrinsics
+- For each referendum with an expired lock, issue a "democracy.removeVote(X)" extrinsic, where X =
```
Comment (laboon) [inline docs/kusama-ledger.md]:
```suggestion
- For each referendum with an expired lock, issue a `democracy.removeVote(X)` extrinsic, where X =
```

File: docs/kusama-ledger.md [hunk 3]
```diff
@@ -197,6 +197,26 @@ want to stake using an account stored on a Ledger device.
...
+  the referendum you voted on and has an expired lock. Make sure you do it from the account with the
+  expired locks - you can't do it from another account.
+- Once you have removed all of the votes, issue a "democracy.unlock(ACCOUNT)" extrinsic, where
```
Comment (laboon) [inline docs/kusama-ledger.md]:
```suggestion
- Once you have removed all of the votes, issue a `democracy.unlock(ACCOUNT)` extrinsic, where
```

**Review by laboon — CHANGES_REQUESTED** [review]

**Comment by salmad3** [comment]
Thanks @laboon, committed the suggestions.

**Pull request merged**